plugins/repository-gcs/src/main/java/org/elasticsearch/repositories/gcs/GoogleCloudStoragePlugin.java

```diff
@@ -112,15 +108,19 @@
         });
     }
 
-    private final Map<String, GoogleCredential> credentials;
+    private final Map<String, GoogleCloudStorageClientSettings> clientsSettings;
```

Comment:
clientsSettings -> clientSettings?

Reply:
There can be multiple client settings keyed by their name so I chose the plural form :)